aibolit/metrics/cognitiveC/cognitive_c.py

```diff
@@ -2,36 +2,64 @@
 from aibolit.patterns.var_middle.var_middle import JavalangImproved
 from typing import List
 
+increment_for = [
+    javalang.tree.IfStatement,
+    javalang.tree.SwitchStatement,
+    javalang.tree.ForStatement,
+    javalang.tree.WhileStatement,
+    javalang.tree.DoStatement,
+    javalang.tree.CatchClause,
+    javalang.tree.BreakStatement,
+    javalang.tree.ContinueStatement,
+    javalang.tree.TernaryExpression,
+    # javalang.tree.BinaryOperation,
+]
+
+nested_for = [
+    javalang.tree.IfStatement,
+    javalang.tree.SwitchStatement,
+    javalang.tree.ForStatement,
+    javalang.tree.WhileStatement,
+    javalang.tree.DoStatement,
+    javalang.tree.CatchClause,
+]
+
 
 class CognitiveComplexity:
     '''
     beta version of Cognitive Complexity
     '''
     def __init__(self):
-        pass
+        self.complexity = 0
```

Comment:
Do not forget that we decided to avoid this attribute and do the same but this value recursively.